src/OSPSuite.Core/Domain/UnitSystem/Unit.cs

```diff
@@ -21,14 +26,57 @@ public class Unit
       /// <summary>
       ///    Name of unit. Should be unique in the dimension containing this unit
       /// </summary>
-      public virtual string Name { get; }
+      public virtual string Name
+      {
+         get => _name;
+         internal set
+         {
+            _name = value;
+            addDefaultUnitSynonyms();
+         }
+      }
+
+      private void addDefaultUnitSynonyms()
+      {
+         addMuSynonyms();
+         addExponentSynonyms();
+      }
+
+      private void addMuSynonyms()
+      {
+         if (!Name.Contains("�"))
+            return;
+
+         AddUnitSynonym(Name.Replace("�", "u"));
+         AddUnitSynonym(Name.Replace("�", "mc"));
+      }
+
+      private void addExponentSynonyms()
+      {
+         if (!(Name.Contains("�") || Name.Contains("�")))
+            return;
+
+         AddUnitSynonym(Name.Replace("�", "2").Replace("�", "3"));
```

Comment:
@Yuri05 I have added this too.
Theroreticallly, there are all the combination possible with u as well etc. and that's going to become quite crazy. so I think we should try to keep it simple and offer some variations but not all possible combinations